lms/templates/certificates/_certificate-supplemental.html

```diff
@@ -0,0 +1,36 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! import mako.runtime %>
+<% mako.runtime.UNDEFINED = '' %>
+
+<aside class="content content-supplemental" role="complimentary"  id="validation-info">
+  <div class="content-supplemental-about">
+    <h2 class="title">${company_about_title}</h2>
+    <div class="copy">
+      <p>${company_about_description}</p>
+    </div>
+    <ul class="list list-actions">
+      <li class="item item-action">
+        <a class="action action-primary action-moreinfo" href="${company_about_url}">${company_about_urltext}</a>
+      </li>
+    </ul>
+  </div>
+
+  <!-- TODO: determine if Open edX needs verification techniques and if so how to support? -->
```

Comment:
Thanks - I've already removed this partial template's reference in the specific web views we've made, so its not being rendered, but I kept it in for archive's sake.

I'm happy to remove it and all of the assets needed if we won't be using it. Give me the final word on that.